salt/kubernetes-common/remove-pre-reboot.sls

```diff
@@ -0,0 +1,12 @@
+# cleanup all the things we have created
+
+/etc/kubernetes/config:
+  file.absent
+
+{% if pillar['cloud']['provider'] == 'openstack' %}
```

Comment:
I think we don't need the `{% if %}`: the `file.absent` will make sure the file is not there, regardless of the cloud provided used...